Source/Resource/Documentation/bext.html

```diff
@@ -54,7 +54,7 @@
 				<ul>
 					<li><p>This element contains the entity responsible for the "archiving" (creation, maintenance, preservation) of this digital item. Entity designations should be as specific as possible including a two-character county code to avoid the potential for conflict in the archiving entity’s name.</p>
 <p>If space permits within the 32 character limit, the archival entity should be identified at the most specific level within the institution.</p>
-<p>Use a standard abbreviation of entity names such as those found in the Guide to Government Acronyms & Abbreviations. If an entity is not on this list, use a familiar abbreviation. Use the standard two-character ISO 3166 alpha 2 country code list.</p>
+<p>Use a standard abbreviation of entity names such as those found in the Guide to Government Acronyms &amp Abbreviations. If an entity is not on this list, use a familiar abbreviation. Use the standard two-character ISO 3166 alpha 2 country code list.</p>
```

Comment:
Sorry, but `&amp;`, including semicolon. See e.g. [W3C example](https://www.w3.org/International/questions/qa-escapes#use)

Reply:
Caught the typo after I commited! Still haven't had enough coffee this morning! Just fixed it